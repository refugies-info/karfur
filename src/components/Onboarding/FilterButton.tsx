import * as React from "react";
import styled from "styled-components/native";
import { RTLTouchableOpacity, RTLView } from "../BasicComponents";
import { theme } from "../../theme";
import { TextSmallBold } from "../StyledText";
import { useTranslationWithRTL } from "../../hooks/useTranslationWithRTL";
import { Icon } from "react-native-eva-icons";

interface Props {
  text: string;
  isSelected: boolean;
  onPress: () => void;
  details?: undefined | string[];
}

const MainContainer = styled(RTLTouchableOpacity)`
  background-color: ${(props: { isSelected: boolean }) =>
    props.isSelected ? theme.colors.lightBlue : theme.colors.white};
  padding: ${theme.margin * 2}px;
  box-shadow: ${(props: { isSelected: boolean }) =>
   props.isSelected ? "none" : "1px 1px 8px rgba(33, 33, 33, 0.24)"};
  border-radius: ${theme.radius * 2}px;
  margin-bottom: ${theme.margin * 3}px;
  justify-content: space-between;
  flex-wrap: wrap;
  elevation: ${(props: { isSelected: boolean }) =>
  props.isSelected ? 0 : 2};
  border-width: 2px;
  border-color: ${(props: { isSelected: boolean }) =>
  props.isSelected ? theme.colors.darkBlue : "transparent"};
`;

const StyledText = styled(TextSmallBold)`
  color: ${(props: { isSelected: boolean }) =>
    props.isSelected ? theme.colors.darkBlue : theme.colors.black};
`;

const RadioButton = styled.View`
  width: 24px;
  height: 24px;
  border-radius: 12px;
  border-width: 1px;
  border-color: ${(props: { isSelected: boolean }) =>
  props.isSelected ? "transparent" : theme.colors.darkGrey};
  background-color: ${(props: { isSelected: boolean }) =>
  !props.isSelected ? "transparent" : theme.colors.darkBlue};
  align-items: center;
  justify-content: center;
`;

export const FilterButton = (props: Props) => {
  const { t } = useTranslationWithRTL();
  return (
    <MainContainer
      onPress={props.onPress}
      isSelected={props.isSelected}
      testID={`test-filter-${props.text}`}
      accessibilityRole="radio"
    >
      <RTLView style={{ justifyContent: "space-between", flex: 1 }}>
        <StyledText isSelected={props.isSelected}>
          {t("filters." + props.text, props.text)}
          {props.details &&
            <StyledText isSelected={props.isSelected}>
            {" "}({props.details.join("/")})
            </StyledText>
          }
        </StyledText>
        <RadioButton isSelected={props.isSelected}>
          {props.isSelected &&
            <Icon
              name="checkmark-outline"
              width={16}
              height={16}
              fill={theme.colors.white}
            />
          }
        </RadioButton>
      </RTLView>
    </MainContainer>
  );
};
