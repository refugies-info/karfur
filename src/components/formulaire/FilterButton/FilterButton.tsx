import React from "react";
import styled from "styled-components/native";
import { TextSmallBold } from "../../StyledText";
import { useTranslationWithRTL } from "../../../hooks/useTranslationWithRTL";
import ChoiceButton from "../ChoiceButton";

export interface FilterButtonProps {
  text: string;
  isSelected: boolean;
  onPress: () => void;
  details?: undefined | string[];
}

const StyledText = styled(TextSmallBold)<{ selected: boolean }>`
  color: ${({ selected, theme }) =>
    selected ? theme.colors.darkBlue : theme.colors.black};
`;

const FilterButton = (props: FilterButtonProps) => {
  const { t } = useTranslationWithRTL();
  return (
    <ChoiceButton
      onPress={props.onPress}
      isSelected={props.isSelected}
      testID={`test-filter-${props.text}`}
    >
      <StyledText isSelected={props.isSelected}>
        {t("filters." + props.text, props.text)}
        {props.text !== "french_level_0" &&
          props.text !== "no_french_level_filter" &&
          props.details && (
            <StyledText isSelected={props.isSelected}>
              {" "}
              ({props.details.join("/")})
            </StyledText>
          )}
      </StyledText>
    </ChoiceButton>
  );
};

export default FilterButton;
