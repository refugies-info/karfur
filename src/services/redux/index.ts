export * from "./SelectedContent";
export * from "./Themes";
export * from "./User";
