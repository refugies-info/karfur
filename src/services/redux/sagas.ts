import { SagaIterator } from "redux-saga";
import { fork } from "redux-saga/effects";
import usersSaga from "./User/user.saga";
import languagesSaga from "./Languages/languages.saga";
import contentsSaga from "./Contents/contents.saga";
import selectedContentSaga from "./SelectedContent/selectedContent.saga";

export function* rootSaga(): SagaIterator {
  yield fork(usersSaga);
  yield fork(languagesSaga);
  yield fork(contentsSaga);
  yield fork(selectedContentSaga);
}
