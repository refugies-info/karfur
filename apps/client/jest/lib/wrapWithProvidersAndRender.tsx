import { TooltipProvider } from "@radix-ui/react-tooltip";
import { render } from "@testing-library/react";
import { initialMockStore } from "__fixtures__/reduxStore";
import React from "react";
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import configureStore from "redux-mock-store";
import { RootState } from "services/rootReducer";

interface WrapWithProvidersAndRenderParams<Props> {
  Component: React.FunctionComponent<any>;
  compProps?: Props;
  reduxState?: Partial<RootState>;
}

/**
 * Provide all the ugly-to-set-up providers for your component to be ready to test
 * @param Component
 * @param compProps properties passed on to [getPropsWithNavigation()] to gen navigation props
 * @param reduxState defaults to initialRootState
 */

export function wrapWithProvidersAndRenderForTesting<T>({
  Component,
  compProps,
  reduxState = initialMockStore,
}: WrapWithProvidersAndRenderParams<T>) {
  const mockStore = configureStore([]);
  const store = mockStore(reduxState);

  const componentWithRedux = (
    <TooltipProvider>
      <Router>
        <Provider store={store}>
          <Component {...compProps} />
        </Provider>
      </Router>
    </TooltipProvider>
  );

  return render(componentWithRedux);
}
