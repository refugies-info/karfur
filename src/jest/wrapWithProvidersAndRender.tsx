import React from "react";
import { render } from "react-native-testing-library";
import { Provider } from "react-redux";
import { createStore } from "redux";
import {
  initialRootStateFactory,
  rootReducer,
  RootState,
} from "../services/redux/reducers";

interface WrapWithProvidersAndRenderParams {
  Component: React.FunctionComponent<unknown>;
  compProps?: Record<string, unknown>;
  reduxState?: Partial<RootState>;
}

/**
 * Provide all the ugly-to-set-up providers for your component to be ready to test
 * @param Component
 * @param compProps properties passed on to [getPropsWithNavigation()] to gen navigation props
 * @param reduxState defaults to initialRootState
 */
export function wrapWithProvidersAndRender({
  Component,
  compProps = {},
  reduxState = initialRootStateFactory(),
}: WrapWithProvidersAndRenderParams) {
  const store = createStore(rootReducer, reduxState);

  const componentWithReduxAndI18n = (
    <Provider store={store}>
      <Component {...compProps} />
    </Provider>
  );

  return render(componentWithReduxAndI18n);
}
