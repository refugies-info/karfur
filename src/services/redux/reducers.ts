import { Action, combineReducers } from "redux";
import {
  loadingStatusReducer,
  LoadingStatusState,
  initialLoadingState,
} from "./LoadingStatus/loadingStatus.reducer";
import {
  languagesReducer,
  LanguageState,
  initialLanguageState,
} from "./Languages/languages.reducer";
import { userReducer, UserState, initialUserState } from "./User/user.reducer";
import {
  contentsReducer,
  ContentsState,
  initialContentsState,
} from "./Contents/contents.reducer";
import {
  SelectedContentState,
  initialSelectedContentState,
  selectedContentReducer,
} from "./SelectedContent/selectedContent.reducer";
import {
  NeedState,
  initialNeedState,
  needsReducer,
} from "./Needs/needs.reducer";
import {
  groupedContentsReducer,
  GroupedContentsState,
  initialGroupedContentsState,
} from "./ContentsGroupedByNeeds/contentsGroupedByNeeds.reducer";
import {
  VoiceOverState,
  initialVoiceOverState,
  voiceOverReducer,
} from "./VoiceOver/voiceOver.reducer";
const appReducer = combineReducers({
  loadingStatus: loadingStatusReducer,
  languages: languagesReducer,
  user: userReducer,
  contents: contentsReducer,
  selectedContent: selectedContentReducer,
  needs: needsReducer,
  groupedContents: groupedContentsReducer,
  voiceOver: voiceOverReducer,
});

export interface RootState {
  loadingStatus: LoadingStatusState;
  languages: LanguageState;
  user: UserState;
  contents: ContentsState;
  selectedContent: SelectedContentState;
  needs: NeedState;
  groupedContents: GroupedContentsState;
  voiceOver: VoiceOverState;
}

type RootReducer = ReturnType<typeof appReducer>;

export const initialRootStateFactory = (): RootState => ({
  loadingStatus: initialLoadingState,
  languages: initialLanguageState,
  user: initialUserState,
  contents: initialContentsState,
  selectedContent: initialSelectedContentState,
  needs: initialNeedState,
  groupedContents: initialGroupedContentsState,
  voiceOver: initialVoiceOverState,
});

export const rootReducer = (
  state: RootState | undefined,
  action: Action<any>
): RootReducer =>
  // @ts-ignore
  appReducer(state, action);
