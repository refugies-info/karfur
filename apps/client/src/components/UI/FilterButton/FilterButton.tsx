import EVAIcon from "@/components/UI/EVAIcon/EVAIcon";
import { cls } from "@/lib/classname";
import React from "react";
import styles from "./FilterButton.module.scss";

interface Props {
  active: boolean;
  className?: string;
  color?: string;
  children?: any;
  [x: string]: any;
}

const FilterButton = React.forwardRef((props: Props, ref) => {
  let { className, active, color, ...bProps } = props;

  const classNames = cls(styles.btn, active && styles.active, !!color && styles.colored, className || "");

  return (
    <button className={classNames} style={color ? { backgroundColor: color } : {}} {...bProps}>
      {props.children}
      {active && <EVAIcon className={styles.close} name="close-outline" fill="white" />}
    </button>
  );
});

FilterButton.displayName = "FilterButton";

export default FilterButton;
