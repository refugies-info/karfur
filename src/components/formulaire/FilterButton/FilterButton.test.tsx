import FilterButton from "./FilterButton";
import { wrapWithProvidersAndRender } from "../../../jest/wrapWithProvidersAndRender";
import { initialRootStateFactory } from "../../../services/redux/reducers";

describe("FilterButton snapshot test suite", () => {
  it("should render without bug", async () => {
    const component = wrapWithProvidersAndRender({
      Component: FilterButton,
      reduxState: {
        ...initialRootStateFactory(),
      },
      compProps: {
        text: "Test choice",
        onPress: () => null,
        isSelected: true,
      },
    });
    expect(component).toMatchSnapshot();
  });

  it("should render not selected", async () => {
    const component = wrapWithProvidersAndRender({
      Component: FilterButton,
      reduxState: {
        ...initialRootStateFactory(),
      },
      compProps: {
        text: "Test choice",
        onPress: () => null,
        isSelected: false,
      },
    });
    expect(component).toMatchSnapshot();
  });

  it("should render with detail", async () => {
    const component = wrapWithProvidersAndRender({
      Component: FilterButton,
      reduxState: {
        ...initialRootStateFactory(),
      },
      compProps: {
        details: ["First detail", "Second detail"],
        text: "Test choice",
        onPress: () => null,
        isSelected: true,
      },
    });
    expect(component).toMatchSnapshot();
  });
});
