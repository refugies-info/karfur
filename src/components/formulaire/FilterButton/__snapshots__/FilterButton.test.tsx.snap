// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterButton snapshot test suite should render not selected 1`] = `
<View
  accessibilityRole="radio"
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#FFFFFF",
      "borderColor": "#FFFFFF",
      "borderRadius": 12,
      "borderWidth": 2,
      "display": "flex",
      "elevation": 6,
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "marginBottom": 24,
      "opacity": 1,
      "paddingBottom": 14,
      "paddingLeft": 14,
      "paddingRight": 14,
      "paddingTop": 14,
      "shadowColor": "#212121",
      "shadowOffset": {
        "height": 1,
        "width": 1,
      },
      "shadowOpacity": 0.16,
      "shadowRadius": 8,
    }
  }
  testID="test-filter-Test choice"
>
  <View
    RTLBehaviour={true}
    animatedStyle={
      {
        "value": {},
      }
    }
    collapsable={false}
    horizontalAlign="flex-start"
    style={
      {
        "alignItems": "center",
        "flexBasis": "auto",
        "flexDirection": "row",
        "flexGrow": 1,
        "justifyContent": "flex-start",
      }
    }
    verticalAlign="center"
  >
    <View
      style={
        [
          {
            "flexBasis": 0,
            "flexGrow": 1,
            "marginRight": 8,
          },
        ]
      }
    >
      <Text
        isRTL={false}
        isSelected={false}
        selectable={true}
        style={
          [
            {
              "color": "#212121",
              "fontFamily": "circularBold",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "left",
            },
          ]
        }
      >
        filters.Test choice
      </Text>
    </View>
    <View
      style={
        [
          {
            "flexBasis": "auto",
            "flexGrow": 0,
          },
        ]
      }
    >
      <View
        selected={false}
        style={
          [
            {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "borderColor": "#5E5E5E",
              "borderRadius": 12,
              "borderWidth": 1,
              "height": 24,
              "justifyContent": "center",
              "width": 24,
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;

exports[`FilterButton snapshot test suite should render with detail 1`] = `
<View
  accessibilityRole="radio"
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#DAE0FB",
      "borderColor": "#0421B1",
      "borderRadius": 12,
      "borderWidth": 2,
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "marginBottom": 24,
      "opacity": 1,
      "paddingBottom": 14,
      "paddingLeft": 14,
      "paddingRight": 14,
      "paddingTop": 14,
    }
  }
  testID="test-filter-Test choice"
>
  <View
    RTLBehaviour={true}
    animatedStyle={
      {
        "value": {},
      }
    }
    collapsable={false}
    horizontalAlign="flex-start"
    style={
      {
        "alignItems": "center",
        "flexBasis": "auto",
        "flexDirection": "row",
        "flexGrow": 1,
        "justifyContent": "flex-start",
      }
    }
    verticalAlign="center"
  >
    <View
      style={
        [
          {
            "flexBasis": 0,
            "flexGrow": 1,
            "marginRight": 8,
          },
        ]
      }
    >
      <Text
        isRTL={false}
        isSelected={true}
        selectable={true}
        style={
          [
            {
              "color": "#0421B1",
              "fontFamily": "circularBold",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "left",
            },
          ]
        }
      >
        filters.Test choice
        <Text
          isRTL={false}
          isSelected={true}
          selectable={true}
          style={
            [
              {
                "color": "#0421B1",
                "fontFamily": "circularBold",
                "fontSize": 16,
                "lineHeight": 20,
                "textAlign": "left",
              },
            ]
          }
        >
           
          (
          First detail/Second detail
          )
        </Text>
      </Text>
    </View>
    <View
      style={
        [
          {
            "flexBasis": "auto",
            "flexGrow": 0,
          },
        ]
      }
    >
      <View
        selected={true}
        style={
          [
            {
              "alignItems": "center",
              "backgroundColor": "#0421B1",
              "borderColor": "transparent",
              "borderRadius": 12,
              "borderWidth": 1,
              "height": 24,
              "justifyContent": "center",
              "width": 24,
            },
          ]
        }
      >
        <View
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "scale": 1,
                },
              ],
            }
          }
        >
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight="16"
            bbWidth="16"
            fill="#FFFFFF"
            focusable={false}
            height={16}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              [
                {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                {
                  "flex": 0,
                  "height": 16,
                  "width": 16,
                },
              ]
            }
            vbHeight={24}
            vbWidth={24}
            width={16}
          >
            <RNSVGGroup
              fill={
                {
                  "payload": 4294967295,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            >
              <RNSVGGroup>
                <RNSVGGroup>
                  <RNSVGRect
                    height="24"
                    opacity={0}
                    width="24"
                    x="0"
                    y="0"
                  />
                  <RNSVGPath
                    d="M9.86 18a1 1 0 0 1-.73-.32l-4.86-5.17a1 1 0 1 1 1.46-1.37l4.12 4.39 8.41-9.2a1 1 0 1 1 1.48 1.34l-9.14 10a1 1 0 0 1-.73.33z"
                  />
                </RNSVGGroup>
              </RNSVGGroup>
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`FilterButton snapshot test suite should render without bug 1`] = `
<View
  accessibilityRole="radio"
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "backgroundColor": "#DAE0FB",
      "borderColor": "#0421B1",
      "borderRadius": 12,
      "borderWidth": 2,
      "display": "flex",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "marginBottom": 24,
      "opacity": 1,
      "paddingBottom": 14,
      "paddingLeft": 14,
      "paddingRight": 14,
      "paddingTop": 14,
    }
  }
  testID="test-filter-Test choice"
>
  <View
    RTLBehaviour={true}
    animatedStyle={
      {
        "value": {},
      }
    }
    collapsable={false}
    horizontalAlign="flex-start"
    style={
      {
        "alignItems": "center",
        "flexBasis": "auto",
        "flexDirection": "row",
        "flexGrow": 1,
        "justifyContent": "flex-start",
      }
    }
    verticalAlign="center"
  >
    <View
      style={
        [
          {
            "flexBasis": 0,
            "flexGrow": 1,
            "marginRight": 8,
          },
        ]
      }
    >
      <Text
        isRTL={false}
        isSelected={true}
        selectable={true}
        style={
          [
            {
              "color": "#0421B1",
              "fontFamily": "circularBold",
              "fontSize": 16,
              "lineHeight": 20,
              "textAlign": "left",
            },
          ]
        }
      >
        filters.Test choice
      </Text>
    </View>
    <View
      style={
        [
          {
            "flexBasis": "auto",
            "flexGrow": 0,
          },
        ]
      }
    >
      <View
        selected={true}
        style={
          [
            {
              "alignItems": "center",
              "backgroundColor": "#0421B1",
              "borderColor": "transparent",
              "borderRadius": 12,
              "borderWidth": 1,
              "height": 24,
              "justifyContent": "center",
              "width": 24,
            },
          ]
        }
      >
        <View
          collapsable={false}
          style={
            {
              "transform": [
                {
                  "scale": 1,
                },
              ],
            }
          }
        >
          <RNSVGSvgView
            align="xMidYMid"
            bbHeight="16"
            bbWidth="16"
            fill="#FFFFFF"
            focusable={false}
            height={16}
            meetOrSlice={0}
            minX={0}
            minY={0}
            style={
              [
                {
                  "backgroundColor": "transparent",
                  "borderWidth": 0,
                },
                {
                  "flex": 0,
                  "height": 16,
                  "width": 16,
                },
              ]
            }
            vbHeight={24}
            vbWidth={24}
            width={16}
          >
            <RNSVGGroup
              fill={
                {
                  "payload": 4294967295,
                  "type": 0,
                }
              }
              propList={
                [
                  "fill",
                ]
              }
            >
              <RNSVGGroup>
                <RNSVGGroup>
                  <RNSVGRect
                    height="24"
                    opacity={0}
                    width="24"
                    x="0"
                    y="0"
                  />
                  <RNSVGPath
                    d="M9.86 18a1 1 0 0 1-.73-.32l-4.86-5.17a1 1 0 1 1 1.46-1.37l4.12 4.39 8.41-9.2a1 1 0 1 1 1.48 1.34l-9.14 10a1 1 0 0 1-.73.33z"
                  />
                </RNSVGGroup>
              </RNSVGGroup>
            </RNSVGGroup>
          </RNSVGSvgView>
        </View>
      </View>
    </View>
  </View>
</View>
`;
